iotawattpy/iotawatt.py

```diff
@@ -140,15 +140,15 @@ def _createOrUpdateSensorSet(
                 suffix=".wh",
                 fromStart=True,
             )
-            self._createOrUpdateSensor(
-                sensors,
-                entity + "_energy",
-                channel_nbr,
-                base_name,
-                type,
-                "WattHours",
-                suffix=".wh",
-            )
+            # self._createOrUpdateSensor(
```

Comment:
what about putting this behind a flag so it can be enabled and we don't lose the functionality ?
and have the HA integration set it to false.